src/components/BookOpts.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const BookOpts = () => (
+  <div className="book-options">
+    <a href="/#">Comments</a>
+    {' | '}
+    <a href="/#">Remove</a>
```

Comment:
- [ ] Let's change this **Remove** link to a placeholder button now as you will use it in the subsequent projects to remove a book. This will merely be a placeholder button. It does not have to work yet.